writers/mongodb/init.go

```diff
@@ -0,0 +1,30 @@
+package mongodb
```

Comment:
Rename this file according to other writers.

Reply:
You don't have this file, and it is consistent with `things` and `users` naming.